README.md

```diff
@@ -1 +1,36 @@
-# DynamicalSystemFramework
\ No newline at end of file
+# DynamicalSystemFramework
+
+The aim of this project is to rework the original [Traffic Flow Dynamics Model](https://github.com/Grufoony/TrafficFlowDynamicsModel).
+This rework consists of a full code rewriting, in order to implement more features (like *intersections*) and get advantage from the latest C++ updates.
+
+## Requirements
+
+By now the project only requires `C++20` and `cmake`.
+To install `cmake`:
```

Comment:
What about Windows, Mac, Arch linux, Fedora ecc?
I think that we can assume that users know how to install CMake, so I would remove this line.